wagtail/contrib/table_block/static/table_block/js/table.js

```diff
@@ -43,6 +45,24 @@ function initTable(id, tableOptions) {
         parentDiv.find('.field-content').width(width);
         parentDiv.find('.fieldname-table .field-content .field-content').width('80%');
     }
+    var setHiddenHeaderValues = function(headerChoice) {
+        if (headerChoice == 'row') {
```

Comment:
Nitpick:

```suggestion
        if (headerChoice === 'row') {
```

Same for all below.